File: JDK11/opentraceability/src/main/java/opentraceability/models/events/kdes/CertificationList.java
package opentraceability.models.events.kdes;

import java.util.*;
import opentraceability.models.common.Certificate;
import opentraceability.utility.attributes.OpenTraceabilityArrayAttribute;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import opentraceability.utility.attributes.OpenTraceabilityObjectAttribute;

public class CertificationList {
    @OpenTraceabilityArrayAttribute(itemType = Certificate.class)
    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute(ns="", name="certification")
    public List<Certificate> certificates = new ArrayList<Certificate>();
}

File: Java/OpenTraceability/src/main/kotlin/opentraceability/models/events/kdes/certificationlist.kt
package models.events.kdes

import models.common.Certificate
import java.util.*
import utility.attributes.*

class CertificationList {
    @OpenTraceabilityArrayAttribute
    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute("","certification")
    var Certificates: ArrayList<Certificate> = ArrayList<Certificate>()
}
